Source/UnitTests/Core/IOS/FS/FileSystemTest.cpp

```diff
@@ -76,9 +76,7 @@ TEST(FileSystem, BasicPathValidity)
 
 TEST(FileSystem, PathSplitting)
 {
-  SplitPathResult result;
-
-  result = {"/shared1", "00000042.app"};
+  SplitPathResult result = {"/shared1", "00000042.app"};
```

Comment:
This reduces readability of the test imo.

Reply:
What is less clear about this?

Reply:
> This reduces readability of the test imo.

I also think the same, as `result` will be assigned different values depending on the test. Leaving the declaration (without initialization) at the beginning makes the intent clearer that this variable might be reused with new values.